integration-tests/overlays/ocp4-stage/configmap.yaml

```diff
@@ -8,3 +8,4 @@ data:
   user-api: "stage.thoth-station.ninja"
   management-api: "management.stage.thoth-station.ninja"
   amun-api: "amun.stage.thoth-station.ninja"
+  tags: "~@seizes_middletier_namespace"
```

Comment:
Note that even v2 tags are documented in the official docs:

https://behave.readthedocs.io/en/latest/tag_expressions.html#tag-expressions-v2

They do not work with [the latest stable release](https://stackoverflow.com/questions/62026928/python-behave-tags-or-logic-does-not-appear-to-be-working). See `--tags-help`:

```
Scenarios inherit tags declared on the Feature level. The simplest
TAG_EXPRESSION is simply a tag::

    --tags @dev

You may even leave off the "@" - behave doesn't mind.

When a tag in a tag expression starts with a ~, this represents boolean NOT::

    --tags ~@dev

A tag expression can have several tags separated by a comma, which represents
logical OR::

    --tags @dev,@wip

The --tags option can be specified several times, and this represents logical
AND, for instance this represents the boolean expression
"(@foo or not @bar) and @zap"::

    --tags @foo,~@bar --tags @zap.

Beware that if you want to use several negative tags to exclude several tags
you have to use logical AND::

    --tags ~@fixme --tags ~@buggy.
```